Makefile

```diff
@@ -110,3 +110,25 @@ cve-report: ## Generate CVE report for a Deckhouse release.
 cve-base-images: ## Check CVE in our base images.
   ##~ Options: SEVERITY=CRITICAL,HIGH
 	./tools/cve/base-images.sh
+
+##@ Update kubernetes control-plane patchversions
+
+.PHONY: update-k8s-patch-versions-deps
+update-k8s-patch-versions-deps: ## Install jq,yq,crane deps for update-patchversion script.
+	make install-jq && make install-yq && make install-crane
```

Comment:
Reusing targets can be one-liner. Great target names, by the way. They are self-explanatory.

```make
.PHONY: update-k8s-patch-versions-deps
update-k8s-patch-versions-deps: install-jq install-yq install-crane
```

Reply:
You are welcome to resolve this thread if you won't feel like changing it.